### PR TITLE
Add a governance file to the CoC document

### PR DESCRIPTION
**Pull request by malvikasharan**
# Summary

- Add the CoC governance file: approved by the committee
- Add log details (and update the previous log details for consistency)
- Get Francois to review and merge this when approved

### CODE REVIEW

File: topic_folders/policies/coc-governance.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+### Incident Response Group’s roles and responsibilities
+
+- An Incident Response Group of a minimum of 3 people are convened for any reported incident
+- An Incident Response Lead is selected in the Incident Response Group to ensure accountability of the report handling
+- The Core Teamstaff liaison will create a collaborative document for each case and share with the Incident Response Group
+- A note-taker will ensure that the notes are maintained for the Incident Response Group and shared with the Code of Conduct committee
+- A time-keeper will notify the Incident Response Group of the various deadlines
+
+*Log detail: This document is drafted by Karin Lagesen, Malvika Sharan, and Karen Cranston and reviewed by the current members of the Code of Conduct committee: Samatha Ahern, Ivo Arrey, Benjamin Schwessinger, Francçois Michonneau and Konrad Förstner (2019-2020) and the Executive Director, Kari L. Jordan.*
```
Comment (malvikasharan) [inline topic_folders/policies/coc-governance.md:109]:
```suggestion
```

**Comment by malvikasharan** [comment]
@fmichonneau can you please review this and merge when ready?

@karenword @konrad can you please confirm if there is anything else that needs updating this PR?

**Comment by maneesha** [comment]
Pinging @fmichonneau  @karenword @konrad 
as @malvikasharan  has asked for review from all of you.

**Comment by malvikasharan** [comment]
Oh, wonderful @maneesha <3 Thank you for pinging the reviewers.

File: topic_folders/policies/coc-governance.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+They may volunteer to rejoin when they are available again following the onboarding procedure described earlier in the document. 
+This will allow us to maintain privacy for the reports that will be handled in their absence and create an opportunity for new members to join the Committee.
+
+The Chairs are expected to sit for approximately one year. 
+New Chairs are selected from among those who have been on the Committee for at least one year. One of the available members can volunteer to be elected to take on the role of a Chair. 
+
+## Main duties of the position holders
+
+All members of the CoCc must read, understand and abide by the [membership agreement](https://docs.carpentries.org/topic_folders/policies/coc-membership-agreement.html#). 
+In addition to the roles and responsibilities outlined in the membership agreement (e.g. engaging in GitHub issues and reviewing transparency reports), committee officers have the g duties outlined below.
```
Comment (fmichonneau) [inline topic_folders/policies/coc-governance.md:60]:
```suggestion
In addition to the roles and responsibilities outlined in the membership agreement (e.g. engaging in GitHub issues and reviewing transparency reports), committee officers have the duties outlined below.
```

File: topic_folders/policies/coc-governance.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- Represent the CoCc in any relevant communication with the EC
+- Relay information to the EC members when required
+- Co-edit and co-release transparency reports with the Incident Response Chair
+- Notify, and request a review if needed, from EC regarding any public communication from CoC committee
+
+### Incident Response Group’s roles and responsibilities
+
+- An Incident Response Group of a minimum of 3 people are convened for any reported incident
+- An Incident Response Lead is selected in the Incident Response Group to ensure accountability of the report handling
+- The Core Teamstaff liaison will create a collaborative document for each case and share with the Incident Response Group
```
Comment (fmichonneau) [inline topic_folders/policies/coc-governance.md:105]:
```suggestion
- The Core Team liaison will create a collaborative document for each case and share with the Incident Response Group
```

File: topic_folders/policies/code-of-conduct.md [hunk 4]
```diff
@@ -73,9 +73,11 @@ Participants who are asked to stop any inappropriate behaviour are expected to c
 
 #### Update Logs
 
-- 2019-07-17 The CoCc membership agreement document was added to the handbook. This document is approved by the members of the Code of Conduct committee after the following process: on 31 January 2019, the first draft of this document was shared with CoCc by Malvika Sharan; from February to June of 2019, this document was intesively reviewed and updated by the committee members Malvika Sharan, Karin Lagesen, Kari L. Jordan, Samantha Ahern, and Simon Waldman; on 18 April 2019, The document was presented to The Carpentries Trainers community.
+- 2020-09-23 The CoCc Governance document was added to the handbook. This document is approved by the members of the Code of Conduct committee, the executive director, Kari L. Jordan and executive committee. Contributors: drafted by Karin Lagesen, Malvika Sharan and Karen Word (2019) and intesively reviewed and updated by the committee members of the Code of Conduct committee: Karin Lagesen, Malvika Sharan, Karen Word, Samatha Ahern, Ivo Arrey, Benjamin Schwessinger, Francçois Michonneau and Konrad Förstner (2019-2020).
```
Comment (fmichonneau) [inline topic_folders/policies/code-of-conduct.md:76]:
```suggestion
- 2020-09-23 The CoCc Governance document was added to the handbook. This document is approved by the members of the Code of Conduct committee, the executive director, Kari L. Jordan and executive committee. Contributors: drafted by Karin Lagesen, Malvika Sharan and Karen Word (2019) and extensively reviewed and updated by the committee members of the Code of Conduct committee: Karin Lagesen, Malvika Sharan, Karen Word, Samatha Ahern, Ivo Arrey, Benjamin Schwessinger, François Michonneau and Konrad Förstner (2019-2020).
```

**Review by fmichonneau — APPROVED** [review]
Thank you for putting this together Malvika. I spotted a few typos but otherwise, it looks great!

**Comment by karenword** [comment]
Hi! I am not sure if I am intended to review this document from the perspective of Instructor Training/ Trainer community, or if I might have been tagged in error (as I share a first name with two committee members). @malvikasharan can you confirm that you have tagged the Karen you are looking for? :)

**Comment by maneesha** [comment]
Looks like @karenword  might have been tagged in error.

@malvikasharan  would you still like @konrad  to review?

**Comment by malvikasharan** [comment]
@karenword I indeed tagged you by mistake, I meant to keep Karen Cranston in loop. please ignore this thread. 🙏

**Comment by malvikasharan** [comment]
@maneesha Francois' review was crucial for this PR. @konrad is aware of this document and has reviewed it in the draft version. If we don't hear from him, it's ok to go ahead and merge it.

**Comment by maneesha** [comment]
Ok, it's been a while so I will merge this now. We can make changes later if needed.

**Pull request merged**